Secretive/Views/StoreListView.swift

```diff
@@ -7,10 +7,19 @@ struct StoreListView: View {
     
     @State private var activeSecret: AnySecret.ID?
     @State private var deletingSecret: AnySecret?
+    @State private var renamingSecret: AnySecret?
 
     @EnvironmentObject private var storeList: SecretStoreList
 
     var body: some View {
+        let secretDeleted = { (secret: AnySecret) in
```

Comment:
Is it possible to do this? Or do you get type checker complaints?

```suggestion
        let secretDeleted = { secret in
```

Reply:
Alternately, might be nicer to just turn these into outright (private) functions.